projects/storefrontapp/src/app/spartacus/features/order/order-feature.module.ts

```diff
@@ -22,6 +33,10 @@ import { OrderRootModule, ORDER_FEATURE } from '@spartacus/order/root';
         },
       },
     }),
+    {
+      provide: MYACCOUNT_V2_ORDER,
+      useFactory: setMyAccountV2OrderToken,
```

Comment:
[NITPICK]
What about simply:
```suggestion
      useValue: environment.myAccountV2
```

Reply:
done